src/Sulu/Bundle/WebsiteBundle/Resources/js/containers/Form/fields/AnalyticsDomainSelect.js

```diff
@@ -26,19 +15,15 @@ class AnalyticsDomainSelect extends React.Component<FieldTypeProps<Array<string>
     };
 
     render() {
-        const {disabled, value} = this.props;
-
-        if (!this.webspace) {
-            return null;
-        }
+        const {disabled, formInspector, value} = this.props;
 
         return (
             <MultiSelect
                 disabled={!!disabled}
                 onChange={this.handleChange}
                 values={value || []}
             >
-                {this.webspace.urls.map(({url}) => (
+                {webspaceStore.getWebspace(formInspector.options.webspace).urls.map(({url}) => (
```

Comment:
same as for the others think would make sense to have `webspaceStore.getWebspace(formInspector.options.webspace).urls` as `@computed` value